tests/testsuite/new.rs

```diff
@@ -305,6 +305,44 @@ fn finds_git_author() {
     assert!(contents.contains(r#"authors = ["foo <gitfoo>"]"#), contents);
 }
 
+#[cargo_test]
+fn finds_git_author_in_included_config() {
+    let included_gitconfig = paths::root().join("foo").join(".gitconfig");
+    included_gitconfig.parent().unwrap().mkdir_p();
+    fs::write(
+        &included_gitconfig,
+        r#"
+        [user]
+            name = foo
+            email = bar
+        "#,
+    )
+    .unwrap();
+
+    let gitconfig = paths::home().join(".gitconfig");
+    fs::write(
+        &gitconfig,
+        format!(
+            r#"
+            [includeIf "gitdir/i:{}"]
+                path = {}
+            "#,
+            included_gitconfig.parent().unwrap().join("").display(),
+            included_gitconfig.display(),
+        )
```

Comment:
Is there anything wrong with how the file contents are formatted? Are there any gotchas that I (as a beginner) should be aware of?

Reply:
I think the Windows error happens because path names have `\` in them instead of `/` which I guess the gitconfig parser is parsing as an escape? I'm not sure how to emit that sort of path to gitconfig myself.